scripts/airmon-ng.linux

```diff
@@ -49,7 +50,7 @@ if [ -n "${3}" ];then
 		CH=3
 	fi
 else
-	CH=10
+	CH=3
```

Comment:
Why is the channel changed?

Reply:
Hi, if we use airmon-ng in pinephone without mentioning channel number then its changed to ch10 according to the shell script and if we change ch10 in rtl8723 then packet injection and mon mode not working. i know this sounds strange but this driver has some issues. that's why I changed the channel number also